activejdbc/src/test/java/org/javalite/activejdbc/ModelTest.java

```diff
@@ -545,12 +546,6 @@ public void shouldGenerateCorrectInsertSQLWithReplacements() {
 
         the(insertSQL).shouldBeEqual("INSERT INTO students (first_name, last_name) VALUES ('Jim', 'O''Connor''s')");
 
-        s.set("dob", getDate(1965, 12, 1));
-
-        insertSQL = s.toInsert("'", "''");
-
-        the(insertSQL).shouldBeEqual("INSERT INTO students (dob, first_name, last_name) VALUES (DATE '1965-12-01', 'Jim', 'O''Connor''s')");
```

Comment:
what is the reason to remove this assertion?

Reply:
Yes, for 2 reasons:
1. The test was failing under M$ _SQLServer_ for the same reason as the test before:
```java
FAILURE! - in org.javalite.activejdbc.ModelTest
shouldGenerateCorrectInsertSQLWithReplacements(org.javalite.activejdbc.ModelTest)  Time elapsed: 0.01 sec  <<< ERROR!
org.javalite.test.jspec.TestException: 
Test object:
java.lang.String == <INSERT INTO students (dob, first_name, last_name) VALUES (CONVERT(date, '1965-12-01'), 'Jim', 'O''Connor''s')>
and expected
java.lang.String == <INSERT INTO students (dob, first_name, last_name) VALUES (DATE '1965-12-01', 'Jim', 'O''Connor''s')>
are not equal, but they should be.
	at org.javalite.activejdbc.ModelTest.shouldGenerateCorrectInsertSQLWithReplacements(ModelTest.java:552
```
> http://ci.javalite.io/job/activejdbc-mssql/126/consoleFull

2. I believ the aim of this test is to verify if the FW properly handles the `'` (apostrofes) in strings, hence the `O'Connor's` name :) For that reason, I didn't find the extra date assertion all that useful...

Do you find that assertion all that important? What's your opinion?
Thanks